src/ctia/http/routes/bulk.clj

```diff
@@ -149,6 +153,60 @@
           {}
           (keys bulk)))
 
+(defn tempids
+  "Merges tempids from all entities
+   {:entity-type1 {:data []
+                   :tempids {transientid1 id1
+                             transientid2 id2}}
+    :entity-type2 {:data []
+                   :tempids {transientid3 id3
+                             transientid4 id4}}}
+
+   ->
+
+   {transientid1 id1
+    transientid2 id2
+    transientid3 id3
+    transientid4 id4}
+
+  The create-entities set the enveloped-result? to True in the flow
+  configuration to get :data and :tempids for each entity in the result."
+  [entities]
+  (reduce (fn [acc [_ v]]
+            (into acc (:tempids v)))
+          {}
+          entities))
```

Comment:
Why not:

~~~
(->> entities
     (map (fn [_ v] (:tempids v))
     (reduce into {}))
~~~